src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -216,6 +230,13 @@ def cli_cosmosdb_update(client,
     if enable_public_network is not None:
         public_network_access = 'Enabled' if enable_public_network else 'Disabled'
 
+    system_assigned_identity = None
```

Comment:
```suggestion
    system_assigned_identity = existing.system_assigned_identity
```
Should we use the original value as default value?